configure-c8y.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+
+if [ `id -u` != 0 ] ; 
+  then printf "Please run as sudo or root!\n"
+  exit
+fi
+printf "\n*********************************************"
+printf "\nThis script lets you setup a thinEdge connection to cumulocity IoT.\nWe assume that the installation was successful and all the necessary services are running.
+Just follow the instructions....."
+printf "\n*********************************************"
+
+printf "\n\n"
+read  -p 'Enter the Cumulocity tenant url e.g. [your-tenant.cumulocity.com]: ' tenant_url
+tenant_url=${tenant_url:-your-tenant.cumulocity.com}
+tedge config set c8y.url $tenant_url
+
+printf "\n\n"
+read -p 'Enter the path to your root certificates. On most Linux systems that is [/etc/ssl/certs]: ' path_root_cert
+path_root_cert=${path_root_cert:-/etc/ssl/certs}
+tedge config set c8y.root.cert.path $path_root_cert
+
+printf "\n\n"
+read -p 'Enter your device-id [my-device]: ' device_id
+device_id=${device_id:-my-device}
+{
+    tedge cert create --device-id $device_id
+} ||
+printf "\n\n"
+read -r -p "Would you like to remove the old cert and create a new one? (y)Yes/(n)No: " response
+case $response in
+    [yY]) 
+        printf "\nRemoving cert...."
+        tedge cert remove
+        tedge cert create --device-id $device_id
+        ;;
+    *) 
+        printf "\nOk bye, enjoy thingEdge.io!!!\n" 
+        exit 
+        ;;    
+esac
+
+
+printf "\n\nYour current configuration:\n"
+tedge config list
+
+
+RET=1
+until [ ${RET} -eq 0 ]; do
```

Comment:
Please use a `while` loop here because as it is more common and the code is not particularly using the `until`.

But that might be my own preference (I had to lookup what an `until` loop was, as I have only used `while` loops before)